bosh-setup/azuredeploy.json

```diff
@@ -66,6 +66,7 @@
   },
   "variables": {
     "apiVersion": "2015-06-15",
```

Comment:
BP for the repo is to use literal string value for the apiVersion.

It is a good practice to use a var in cases like this but unfortunately the way the json schemas are designed the document can't be evaluated if an expression is used.  It affects all JSON schema validations (editors, rest api's, etc).